nilearn/interfaces/fmriprep/tests/test_load_confounds.py

```diff
@@ -51,14 +52,9 @@ def _simu_img(tmp_path, demean):
 
     # create random noise and a random mixture of confounds standardized
     # to zero mean and unit variance
-    if sys.version_info < (3, 7):  # fall back to random state for 3.6
-        np.random.RandomState(42)
-        beta = np.random.rand(nx * ny * nz, X.shape[1])
-        tseries_rand = scale(np.random.rand(nx * ny * nz, nt), axis=1)
-    else:
-        randome_state = np.random.default_rng(0)
-        beta = randome_state.random((nx * ny * nz, X.shape[1]))
-        tseries_rand = scale(randome_state.random((nx * ny * nz, nt)), axis=1)
+    rng = _rng()
+    beta = rng.random((nx * ny * nz, X.shape[1]))
+    tseries_rand = scale(rng.random((nx * ny * nz, nt)), axis=1)
```

Comment:
Regarding this, we should actually upgrade the fixture to use `np.random.default_rng`

Reply:
indeed